src/OneExplorer.ts

```diff
@@ -152,29 +177,131 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     }
   }
 
+  // TODO(dayo) extract file-relative functions as another module
+  private parseInputPath = (configPath: string, modelPath: string): string|undefined => {
+    const config = readIni(configPath);
+    const ext = path.extname(modelPath).slice(1);
+
+    if (ext === undefined || config === null) {
+      return undefined;
+    }
+
+    return config[`one-import-${ext}` as keyof typeof config] ?
+        config[`one-import-${ext}` as keyof typeof config]['input_path'] :
+        undefined;
+  };
+
+  // TODO(dayo) extract file-relative functions as another module
+  private grepTargetInCommand = (str: string): string[] => {
+    let targets: string[] = [];
+    for (let entry of str.split(' ')) {
+      if (path.extname(entry) === '.tvn' || path.extname(entry) === '.circle') {
+        targets.push(entry);
+      }
+    }
+    return targets;
+  };
+
+  // TODO(dayo) extract file-relative functions as another module
+  private grepTarget = (str: string): string[] => {
+    let targets: string[] = [];
+    // TODO(dayo) add checks
+    targets.push(str);
+    return targets;
```

Comment:
1. why don't you use `return [str]` ?
2. out of curiosity, could you write a comment why the name is `grepTarget`?

Reply:
1. sure.
2. I thought this function would do grep something as grepTargetInCommand but it actually doesn't. I can replace them with a simple array function, but I didn't want to repeat them. Maybe `grepAll` would do?